docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -610,25 +565,24 @@ The fact that the DDL procedure in TiDB is mature helps to achieve some features
 - DDL is rollbackable as the middle states can transform from one to another
```

Comment:
Old line 517, 576, 581 should not the table level PLACEMENT POLICY be before the PARTITION BY syntax?